internal/app/commands/cloud/auth/auth_flow.go

```diff
@@ -149,6 +193,30 @@ func oAuthGetTokens(flow *Flow) (*tokens, error) {
 	return nil, fmt.Errorf("authentication flow expired")
 }
 
+func oAuthRefreshTokens(refreshToken string) (*tokens, error) {
+	var res refreshTokenRequest
```

Comment:
Did you mean refreshTokenResponse ?

Reply:
Correct, I didn't see another one.
either rename both to something abstract like `data` if they are used for both request and response data,
or if they only used for response, then rename them to response